docker/build/assets.Dockerfile

```diff
@@ -226,9 +226,11 @@ RUN echo "Patching up wheel using auditwheel..." && \
         --exclude libcublasLt.so.11 \
         --exclude libcurand.so.10 \
         --exclude libcusolver.so.11 \
+        --exclude libcusparse.so.11 \
```

Comment:
good catch - both of these should be excluded. However, I think you need to add cusparse to the python wheel dependencies in the toml file then. You will also need to make a suitable update in a bunch of other places - search for curand, and you need to do probably the same thing for cusparse across the board. It would be good to make that a different PR.

Reply:
part of a separate PR